test/data/zoomify/ImageProperties.xml

```diff
@@ -0,0 +1 @@
+<IMAGE_PROPERTIES WIDTH="1000" HEIGHT="1000" NUMTILES="21" NUMIMAGES="1" VERSION="1.8" TILESIZE="256" />
```

Comment:
I see this file includes NUMTILES, NUMIMAGES, and VERSION, which you're not passing in your example in the test. Are those pieces of info unnecessary? 

Also, I assume this is a standard Zoomify XML, but it's interesting to me that it's missing some of the standard XML bits like `<?xml version="1.0" encoding="UTF-8"?>`...is that standard for Zoomify?